### PR TITLE
An article about agile in the context of a covid model

### PR DESCRIPTION
**Pull request by bobturneruk**

### CODE REVIEW

**Comment by bobturneruk** [comment]
I will need Jess to OK this before it goes out, but would appreciate some feedback from the team. Maybe a bit introspective and best confined to my hard drive.

File: _posts/2020-09-03-agile.md [hunk 1]
```diff
@@ -30,50 +30,40 @@ Our software was one of a number of projects under the umbrella of SCRC. The ide
...
 
-Personally, I was worried about my lack of experience at the start of the project - however it turns out that years of interdisciplinary research working with PIs, students, technicians, postdocs in chemistry, physics engineering, medicine, biology yielded some useful transferrable skills. I would like to know more about how to do "agile" and "Scrum" properly - maybe I've missed something really important? Would a proper "Scrum-Master" think this blog post laughable? But I'm minded by my experience of studying management (albeit 15 years ago) - management models are rarely based on quantitative evidence in the way that scientific models are. There's no proof they're going to work in a particular situation.
+A challenge for the future is how to reconcile the adaptability of these methods, which are great at responding to quick changes in needs, with the requirement to hit specific milestones on particular deadlines in an academic environment where a projects resources are capped years in advance.
```
Comment (davidwyld) [inline _posts/2020-09-03-agile.md]: projects > project's

File: _posts/2020-09-03-agile.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+Personally, I was worried about my lack of experience at the start of the project - however it turns out that years of interdisciplinary research working with PIs, students, technicians, postdocs in chemistry, physics engineering, medicine, biology yielded some useful transferrable skills. I would like to know more about how to do "agile" and "Scrum" properly - maybe I've missed something really important? Would a proper "Scrum-Master" think this blog post laughable? But I'm minded by my experience of studying management (albeit 15 years ago) - management models are rarely based on quantitative evidence in the way that scientific models are. There's no proof they're going to work in a particular situation.
```
Comment (davidwyld) [inline _posts/2020-09-03-agile.md]: physics, engineering

Comment (davidwyld) [reply _posts/2020-09-03-agile.md]: and biology

**Comment by magicicada** [comment]
Looks fine to me - interesting for me to hear your reflection on how it all went.

**Comment by bobturneruk** [comment]
Thanks @magicicada and @davidwilby !

File: _posts/2020-09-03-agile.md (hunk 1 above)
Comment (bobturneruk) [reply _posts/2020-09-03-agile.md]: Ta - changed.

File: _posts/2020-09-03-agile.md (hunk 2 above)
Comment (bobturneruk) [reply _posts/2020-09-03-agile.md]: Ta!

**Pull request merged**